### PR TITLE
Feature/df 103

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: src/main/webapp/app/home/tools/job-publication-tool/job-publication-tool.component.ts [hunk 1]
```diff
@@ -452,20 +372,20 @@ export class JobPublicationToolComponent implements OnInit, OnDestroy {
...
-                firstName: [formModel.publicContact.firstName],
-                lastName: [formModel.publicContact.lastName],
+                salutation: [formModel.publicContact.salutation, Validators.required],
+                firstName: [formModel.publicContact.firstName, Validators.required],
+                lastName: [formModel.publicContact.lastName, Validators.required],
                 phoneNumber: [formModel.publicContact.phoneNumber],
-                email: [formModel.publicContact.email],
-            }),
+                email: [formModel.publicContact.email, Validators.pattern(EMAIL_REGEX)],
+            }, { validator: atLeastOneRequired(Validators.required, ['salutation', 'firstName', 'lastName']) }),
```
Comment (pavax) [inline src/main/webapp/app/home/tools/job-publication-tool/job-publication-tool.component.ts]: where do we check that phoneNumber OR email has to be required?

File: src/main/webapp/app/home/tools/job-publication-tool/job-publication-tool.component.ts [hunk 2]
```diff
@@ -381,9 +301,9 @@ export class JobPublicationToolComponent implements OnInit, OnDestroy {
     }
 
     private createJobPublicationForm(formModel: JobPublicationForm): FormGroup {
-        const atLeastOneRequired = (validator: ValidatorFn, excludeControl: string) => (group: FormGroup): ValidationErrors | null => {
+        const atLeastOneRequired = (validator: ValidatorFn, excludeControl: string[]) => (group: FormGroup): ValidationErrors | null => {
             const hasAtLeastOne = group && group.controls && Object.keys(group.controls)
-                .filter((controlName) => controlName !== excludeControl)
+                .filter((controlName) =>  excludeControl.includes(controlName))
```
Comment (birom) [inline src/main/webapp/app/home/tools/job-publication-tool/job-publication-tool.component.ts]: This must be
```
 .filter((controlName) => !excludeControl.includes(controlName))
```

Comment (fabiangotzen) [reply src/main/webapp/app/home/tools/job-publication-tool/job-publication-tool.component.ts]: I have decided to flip it instead so we send the includedControls to make it less misleading: 

 validator: atLeastOneRequired(Validators.required, ['phoneNumber', 'email']) }),

.filter((controlName) =>  includedControls.includes(controlName))

Comment (birom) [reply src/main/webapp/app/home/tools/job-publication-tool/job-publication-tool.component.ts]: Ok, it is also good to me. Thx.

**Pull request merged**